ubyssey/templates/widgets/frontpage/default.html

```diff
@@ -8,10 +8,10 @@
         {% if articles.trending %}
           <div class="o-trending-article">
           {% with articles.trending as trending_article %}
-              <div>
+          <span>
```

Comment:
Not a blocker but indent is bit off here

Reply:
ill fix it before i merge